lib/project_types/script/config/extension_points.yml

```diff
@@ -1,24 +1,24 @@
 discount:
   assemblyscript:
     package: "@shopify/extension-point-as-discount"
-    version: "^0.2.3"
+    version: "^0.2.4"
     sdk-version: "^6.0.0"
     toolchain-version: "^1.1.0"
 unit_limit_per_order:
   assemblyscript:
     package: "@shopify/extension-point-as-unit-limit-per-order"
-    version: "^0.1.4"
+    version: "^0.1.6"
```

Comment:
That went up 2 versions?  🤷‍♀️

Reply:
Before my change for this package, there was a hotfix PR https://github.com/Shopify/extension-points/pull/119 that bumped it to 0.1.5 which didn't update CLI.